ci/terraform/shared/authdev2.tfvars

```diff
@@ -6,30 +6,7 @@ password_pepper     = "fake-pepper"
 enable_api_gateway_execution_request_tracing = true
 di_tools_signing_profile_version_arn         = "arn:aws:signer:eu-west-2:706615647326:/signing-profiles/di_auth_lambda_signing_20220214175605677200000001/ZPqg7ZUgCP"
 
-stub_rp_clients = [
```

Comment:
this leaves authdev1/authdev2 with no entries, but I think @ethanmills is on this?

Reply:
Yup